packages/core/src/components/icon/icon.tsx

```diff
@@ -25,23 +25,26 @@ export interface IIconProps extends IIntentProps, IProps {
     color?: string;
 
     /**
-     * Name of a Blueprint UI icon, or an icon element, to render.
-     * This prop is required because it determines the content of the component, but it can
+     * Name of a Blueprint UI icon, or an icon element, to render. This prop is
+     * required because it determines the content of the component, but it can
      * be explicitly set to falsy values to render nothing.
      *
-     * - If `null` or `undefined` or `false`, this component will render nothing.
-     * - If given an `IconName` (a string literal union of all icon names),
-     *   that icon will be rendered as an `<svg>` with `<path>` tags.
-     * - If given a `JSX.Element`, that element will be rendered and _all other props on this component are ignored._
-     *   This type is supported to simplify usage of this component in other Blueprint components.
-     *   As a consumer, you should never use `<Icon icon={<element />}` directly; simply render `<element />` instead.
+     * - If `null` or `undefined` or `false`, this component will render
+     *   nothing.
+     * - If given an `IconName` (a string literal union of all icon names), that
+     *   icon will be rendered as an `<svg>` with `<path>` tags. Unknown strings
+     *   will render a blank icon.
```

Comment:
is a "blank icon" different than an empty `<svg>`? if so, how? if not, how is it an "icon" at all?

the wording makes it sound like there is some sort of stand in icon for a "blank" situation, kind of like the broken `<img>` graphic.

Reply:
![image](https://user-images.githubusercontent.com/464822/45443555-e8554300-b679-11e8-9833-0ff3c12a4741.png)
😂